src/test_scenes/sceneFilePicker.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
```

Comment:
Shouldn't we import components from magic-script-components rather than providing lowcase jsx notation?

Reply:
This should be part the of a separate change. It's legacy code and is a major change for the entire catalog.